tests/test_repro.py

```diff
@@ -621,7 +621,77 @@ def scheme_sep(self):
     def sep(self):
         return '/'
 
-    def test(self):
+    def import_already_cached(self):
+        stage = self.dvc.imp(self.BAR, 'imported')
```

Comment:
You are still only testing a local dependency. You should use https://github.com/iterative/dvc/blob/0.23.2/tests/test_repro.py#L661 instead.